packages/vscode-extension/package.json

```diff
@@ -533,8 +533,8 @@
         "icon": "$(run)"
       },
       {
-        "command": "fx-extension.cicdGuide",
-        "title": "%teamstoolkit.commands.cicd.title%",
+        "command": "fx-extension.addCICDWorkflows",
+        "title": "Teams: Add CI/CD Workflows",
```

Comment:
Please remove the title string to package.nls.json as other commands since I'm currently working on localization and this is needed for onboarding to localization team.

Reply:
Good to know that - let me check it.

Reply:
Done @HuihuiWu-Microsoft